internal/proxy/multi_rate_limiter.go

```diff
@@ -49,80 +51,83 @@ func GetQuotaErrorString(errCode commonpb.ErrorCode) string {
 // MultiRateLimiter includes multilevel rate limiters, such as global rateLimiter,
 // collection level rateLimiter and so on. It also implements Limiter interface.
 type MultiRateLimiter struct {
-	globalRateLimiter *rateLimiter
 	// TODO: add collection level rateLimiter
-	quotaStatesMu sync.RWMutex
-	quotaStates   map[milvuspb.QuotaState]commonpb.ErrorCode
+	quotaStatesMu      sync.RWMutex
+	collectionLimiters map[string]*rateLimiter
```

Comment:
why not use collectionID as the key?

Reply:
> why not use collectionID as the key?

cause request  in proxy only contains collection name, check with collection_name is a lower cost choice